.gitignore

```diff
@@ -74,6 +74,9 @@ typings/
 /docs/.cache-loader
 
 # Built Go Binaries
+platform-linux
+platform.exe
+platform-macos
```

Comment:
These should be removed

Reply:
Keep them in incase anyone still needs to use v1...